src/primitives/common/plane_sector.rs

```diff
@@ -24,10 +32,19 @@ enum Operation {
 /// half-planes.
 #[derive(Copy, Clone, PartialEq, PartialOrd, Debug)]
 pub struct PlaneSector {
+    /// Origin of the two half planes.
+    ///
+    /// The origin of the half planes is equal to the center point of the arc or circle scaled
+    /// up by a factor of 2.
+    // TODO: make private
+    pub(crate) origin: Point,
```

Comment:
I think this still needs to be some sort of `pub`:

```suggestion
    pub(in crate::primitives) origin: Point,
```

Reply:
This will be changed in the next PR and the field might be removed completely.